hack/generator/pkg/config/configuration.go

```diff
@@ -149,3 +162,30 @@ func (config *Configuration) TransformType(name astmodel.TypeName) (astmodel.Typ
 	// No matches, return nil
 	return nil, ""
 }
+
+// StatusConfiguration provides configuration options for the
+// status parts of resources, which are generated from the
+// Azure Swagger specs.
+type StatusConfiguration struct {
+	// The root URL of the status (Swagger) files (relative to this file)
+	SchemaRoot string `yaml:"schemaRoot"`
+
+	// The per-namespace configuration
+	Schemas []StatusNamespace `yaml:"schemas"`
+}
+
+// StatusNamespace provides per-namespace configuration
+type StatusNamespace struct {
```

Comment:
oh, it _is_ a namespace (I had made a comment earlier related to this)...

Maybe there's a better name for `StatusConfiguration.Schemas` then? Does `StatusConfiguration.Namespaces` work better...?

Reply:
Right so we have a bit of a conflict with the ARM and Kubernetes naming here. I think I'm going to drop the ARM naming because I'm not going to end up using much of their config file.

Reply:
Renamed this area.